simple_sso/sso_server/server.py

```diff
@@ -1,19 +1,29 @@
+import datetime
 from urllib.parse import urlparse, urlencode, urlunparse
 
 from django.contrib import admin
 from django.contrib.admin.options import ModelAdmin
-from django.http import (HttpResponseForbidden, HttpResponseBadRequest, HttpResponseRedirect, QueryDict)
-from django.urls import reverse
+from django.contrib.sessions.models import Session
+from django.http import HttpResponse, HttpResponseForbidden, HttpResponseBadRequest, HttpResponseRedirect, QueryDict
 from django.urls import re_path
+from django.urls import reverse
 from django.utils import timezone
 from django.views.generic.base import View
 from itsdangerous import URLSafeTimedSerializer
+from simple_sso.settings import settings
 from simple_sso.sso_server.models import Token, Consumer
-import datetime
 from webservices.models import Provider
 from webservices.sync import provider_for_django
 
 
+class ThrowableHttpResponse(Exception):
```

Comment:
This seems to be here to capture and ignore errors (great for debugging but not production.) Please let me know if I'm missing something!